packages/app-mobile/components/side-menu-content.tsx

```diff
@@ -146,11 +147,30 @@ const SideMenuContentComponent = (props: Props) => {
 
 		const folder = folderOrAll as FolderEntity;
 
-		if (folder && folder.id === getTrashFolderId()) return;
-
 		const menuItems: any[] = [];
 
-		if (folder && !!folder.deleted_time) {
+		if (folder && folder.id === getTrashFolderId()) {
+			menuItems.push({
+				text: _('Empty trash'),
+				onPress: async () => {
+					Alert.alert('', _('Are you sure you want to empty the trash?\n\nAll notes and sub-notebooks within trash will be lost forever'), [
```

Comment:
Please use the exact same message as in the desktop app: `This will permanently delete all items in the trash. Continue?`